apps/webstore/src/app/app-routing.module.ts

```diff
@@ -31,6 +32,11 @@ const routes: Routes = [
     component: VerificationComponent,
     canActivate: [CookieGuard],
   },
+  {
+    path: 'unsubscribe/:token',
+    component: UnsubscribeComponent,
+    canActivate: [CookieGuard],
+  },
```

Comment:
Should be `newsletter/unsubscribe:newsletterId` to separate it from other functionality
Keep in mind that this is a `newsletterId`, not a `token`, so name it accordingly
_warning: untested code_
```suggestion
  {
    path: 'newsletter',
    children: [
      {
        path: 'unsubscribe/:newsletterId',
        component: UnsubscribeComponent,
        canActivate: [CookieGuard],
      },
    ],
  },
```